Code/Editor/EditorFramework/Document/Implementation/GameObjectDocument.cpp

```diff
@@ -308,10 +308,11 @@ void ezGameObjectDocument::DetermineNodeName(const ezDocumentObject* pObject, co
 
     for (auto pProperty : properties)
     {
+      const auto type = pProperty->GetSpecificType();
+
       // search for string properties that also have an asset browser property -> they reference an asset, so this is most likely the most
       // relevant property
-      if (
```

Comment:
Bugfix for the resource names not showing up anymore in the scenegraph window as node names